containers.conf

```diff
@@ -6,6 +6,10 @@ cgroup_conf=[
 	"memory.oom.group=1",
 ]
 
+# Temporary default to host network until we fix private network bridge setup
+# when the qm container doesn't unmask all the virtual filesystems.
+netns="host"
```

Comment:
isnt it another wording for podman run --net=host inside QM?
so there was that one for subnets 
see L#20